### ISSUE TITLE
Getting TART calibration data via cmd failed with new error

### ISSUE THREAD

**Issue by rubyvanrooyen** [tart-telescope/notebooks#7]
Seems to have a small error that is raised when running example command:   
`!tart_calibration_data --api {API} --file {CALIB_DATA} --n 1 --i 25`

```
Loading new data from https://tart.elec.ac.nz/signal/...
Traceback (most recent call last):
  File "/usr/local/bin/tart_calibration_data", line 85, in <module>
    vis_json, src_json = load_data(api, config)
  File "/usr/local/bin/tart_calibration_data", line 34, in load_data
    print(f"Getting catalog from {api.catalog_url(config, datestr=ts.isoformat())}")
TypeError: catalog_url() missing 1 required positional argument: 'lat
```

Checked that newest version of all software was installed, see notebook for full workflow
[TART_cmd_calibrate.ipynb](https://github.com/tart-telescope/notebooks/blob/main/getting_started/TART_cmd_calibrate.ipynb)

**Comment by rubyvanrooyen** [comment]
Note to me: This will also affect example notebook:
[TART_cmd_tools.ipynb](https://github.com/tart-telescope/notebooks/blob/main/calibration/phase/TART_cmd_tools.ipynb)

**Comment by tmolteno** [comment]
Should be a quick fix (hopefully)!

**Comment by tmolteno** [comment]
Should be fixed in version 1.1.2b6 of tart_tools.
